constants/currency.ts

```diff
@@ -89,4 +89,6 @@ export const FIAT_SYNTHS = new Set([
 	Synths.sCHF,
 ]);
 
+export const ETH_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
```

Comment:
maybe make note it is just a random eth address or a placeholder eth address instead of just naming it eth address?

Reply:
I think this is the ETH address that 1inch uses, but yeah maybe `ONEINCH_ETH_ADDRESS` or something